boxfish-studio/iota-wiki
# feat: inherit docusaurus configuration from core repo

# Description of change

Remove previous counfiguration and use cdocusaurus configuration form the core repo.

## Links to any relevant issues

Fixes #20 

## Testing steps:

1. Change the config in `common/docusaurus.config.js` and set trailingSlash to `false`.
2. Run `cd cli && yarn build && cd ..` to generate necessary files.
3. Run `yarn add wiki-cli@link:./cli` to link the cli package to the wiki.
4. Run `node ./node_modules/wiki-cli/dist/cli/src/index.js` to execute the cli. (this is because the package is linked locally)
5. Select with the arrow the first option `> Preview the current directory`.
6. Navigate through the wiki. Trailing Slash should be disabled.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
